packages/vite/src/client/client.ts

```diff
@@ -172,9 +172,10 @@ async function queueUpdate(p: Promise<(() => void) | undefined>) {
 socket.addEventListener('close', ({ wasClean }) => {
   if (wasClean) return
   console.log(`[vite] server connection lost. polling for restart...`)
-  setInterval(() => {
+  const interval = setInterval(() => {
     fetch(`${base}__vite_ping`)
```

Comment:
I suppose this solution is not perfect if the fetch takes longer than the interval... but we can maybe ignore that very unlikely possibility (chain setTimeout would be safer)

Reply:
Yeah I think it's better to refactor it to use chained `setTimeout` instead.

Reply:
Done.